src/lib/y2partitioner/widgets/md_devices_selector.rb

```diff
@@ -128,6 +132,25 @@ def handle(event)
       # @return [Y2Partitioner::Actions::Controllers::Md]
       attr_reader :controller
 
+      def help_available
+        _("<p><b>Available Devices:</b> " \
+          "Unused disks and partitions that can be used for a RAID. " \
```

Comment:
Right now, the MD RAID can only be created using partitions, so the description should be adapted. See https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2partitioner/actions/controllers/md.rb#L75

Reply:
But that's a bug then, right? Wouldn't I really want to use plain disks for something like mirroring? If this is a limitation of the current code, will somebody adapt the help text when this limitation is lifted some day? (In my experience, this always gets forgotten).

Reply:
BTW Arvin confirmed that libstorage-ng does not have that restriction, so this is purely a restriction of our yast-storage-ng Ruby code.

Reply:
Probably it only uses partitions because it is what the old partitioner did. Ok, I am fine if we keep the current help text, but we also should create a new card in trello to correct the behavior of the code. Otherwise, this mismatch will come with bug reports, I guess.